packages/jaeger-ui/src/utils/readJsonFile.tsx

```diff
@@ -32,8 +40,6 @@ export default function readJsonFile(fileList: { file: File }): Promise<string>
             .catch(() => {
               reject(new Error(`Error converting traces to OTLP`));
             });
-        } else {
-          resolve(traceObj);
```

Comment:
why is this removed?

Reply:
I removed it during debugging!!
Adding it now!